.eslintrc.json

```diff
@@ -108,6 +108,9 @@
         },
         {
             "files": ["*.ts", "*.tsx"],
+            "parserOptions": {
+                "project": ["./tsconfig(.*)?.json"]
+            },
             "extends": ["plugin:@nrwl/nx/typescript"],
             "rules": {
                 "@typescript-eslint/no-var-requires": "off",
```

Comment:
we need to identify which of these rules require a parserOptions.project and move them out of here and into the project.

Reply:
So I moved out `"@typescript-eslint/no-floating-promises": "error",` from here and put into next init generator eslintrc. The rest run fine as is from my understanding (not getting that parsing related error)